cypress/e2e/Payment.Game.Stopped.js

```diff
@@ -0,0 +1,109 @@
+// Experiment.Step.Through.js created with Cypress
```

Comment:
Lets make sure the important parts of this are in the Games_Filled and Batch_Canceled tests and remove this extra test.